lib/push.py

```diff
@@ -82,19 +84,51 @@ def manifest_new(class_):
 
    def cleanup(self):
       ch.INFO("cleaning up")
-      # Delete the tarballs since we can't yet cache them.
-      for (_, tar_c) in self.layers:
-         ch.VERBOSE("deleting tarball: %s" % tar_c)
-         tar_c.unlink_()
-
-   def prepare(self):
-      """Prepare self.image for pushing to self.dst_ref. Return tuple: (list
-         of gzipped layer tarball paths, config as a sequence of bytes,
-         manifest as a sequence of bytes).
-
-         There is not currently any support for re-using any previously
-         prepared files already in the upload cache, because we don't yet have
-         a way to know if these have changed until they are already build."""
+     # # Delete the tarballs since we can't yet cache them.
+     # for (_, tar_c) in self.layers:
+     #    ch.VERBOSE("deleting tarball: %s" % tar_c)
+     #    tar_c.unlink_()
+
+   def get_prev_prepared(self):
```

Comment:
```suggestion
   def prepare_existing(self):
```

In general, we try to avoid starting methods with the verb `get`. This is because: 1) we typically order methods alphabetically, and 2) we try to maintain method locality for related methods.

In this case the verb 'prepare' is common for both methods, so we should name the function something like `prepare_existing`. This keeps the methods in alphabetical order.